packages/portal-subgraph/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "@graphprotocol/graph-ts/types/tsconfig.base.json",
+  "include": ["src"]
```

Comment:
looks good!

minor, but is it feasible to use `npm` instead of `yarn`? re `yarn.lock` below. good to minimize tools & dependencies

Reply:
removed!